security/docker/istio_ca.crt

```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think you can remove these files if you want to dynamically generate the certificates in the e2e tests.

Reply:
"bazel build ..." command will be failed without initial files.

Reply:
I see. Can you put the certs into cert/ directory, like this:
https://github.com/istio/istio/tree/master/pilot/docker/certs